Include/moduleobject.h

```diff
@@ -100,6 +100,7 @@ typedef struct {
     PyObject *lz_obj;
     PyObject *lz_next;
     int lz_resolving;
+    int lz_eager_loaded;
```

Comment:
it's not clear to me why we need this new flag on `PyLazyImport`. It seems like it is basically only used as a local variable in `PyImport_LoadLazyImport`.